### ISSUE TITLE
Wrong qrcode-generator version in bower.json

### ISSUE THREAD

**Issue by nagistaja** [monospaced/angular-qrcode#60]
bower qrcode-generator#v0.2.0       ENORESTARGET No tag found that was able to satisfy v0.2.0

Additional error details:
Available versions in git://github.com/monospaced/bower-qrcode-generator.git: 0.1.0, 0.0.3, 0.0.2, 0.0.1, 0.0.0

**Comment by monospaced** [comment]
Thanks for the pick up 👍 

Should be fixed now, can you let me know?

**Comment by nagistaja** [comment]
Bower still fails as it can't find a dependency without git url 
"qrcode-generator": "git://github.com/monospaced/bower-qrcode-generator.git#v0.1.0" 

```
bower ECONFLICT Unable to find suitable version for qrcode-generator

Build step 'Execute shell' marked build as failure
```

And looks like qrcode-generator team just forgot to add a tag for last version.

**Comment by monospaced** [comment]
I did tag the release after seeing your issue.

It's here:

https://github.com/monospaced/bower-qrcode-generator/releases/tag/v0.2.0

https://github.com/monospaced/angular-qrcode/blob/master/bower.json#L13

Still no luck?

BTW `qrcode-generator` does not support bower itself, which is why I made the intermediary repo `bower-qrcode-generator`. Bower usage of `angular-qrcode` is essentially deprecated, no longer mentioned in README, but I was hoping to keep it alive those still using.
